CHANGELOG.md

```diff
@@ -13,8 +13,22 @@ Given a version number MAJOR.MINOR.PATCH, increment:
 
 
 ## [Unreleased]
+### Changed
+- amount to nominal_amount fine_amount to fine, interest_amount to interest and discount_amount to discounts on response_due method of DynamicBrcode resource
```

Comment:
amount to nominal_amount, fine_amount to fine, interest_amount to interest, and discount_amount to discounts on response_due method of DynamicBrcode resource